nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
```

Comment:
- [ ] Following the project requirements, you should create a Decorator class that it inherits from Nameable.
Decorator class:
- In the constructor assign a nameable object from params to an instance variable.
- Implement the correct_name method that returns the result of the correct_name method of the @nameable.